### ISSUE TITLE
Provide way for users to disable `accessibility.verbosity` setting from within help menu

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#185127]
They should be able to press a key to toggle this on and off

**Comment by meganrogge** [comment]
Turn on a screen reader. Focus the terminal - you'll hear a hint about how to access the help menu. 

Open the accessibility help menu - for example, in the terminal, use `alt+f1`. 

See message about how to disable the verbosity setting. Follow the instructions.

Close the help menu, focus the terminal. you shouldn't hear that hint. Open the help menu. You shouldn't see that message.

**Comment by meganrogge** [comment]
that's another bug
